health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/BoundarySelection.js

```
@@ -111,38 +111,38 @@ const BoundarySelection = ({ onSelect, props: customProps, ...props }) => {
-                onClick={() => {
-                  setShowPopup(false);
-                //   setCanUpdate(true);
-                }}
-              />
-            ]}
-            // sortFooterChildren={true}
-          ></PopUp>}
+          />
+        ]}
+      // sortFooterChildren={true}
+      ></PopUp>}
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)